src/ensembl/src/header/Header.tsx

```diff
@@ -75,6 +76,7 @@ export const Header = () => {
     <header>
       <Topbar />
       <Launchbar />
+      <CommunicationPanel />
```

Comment:
What if we move this to the communication button? It would at least be a more logical place for this component than the `Header`.

Reply:
That's done